.gitlab-ci.yml

```diff
@@ -104,37 +104,26 @@ changelog:
     # deal with the component's changelogs
     - |
       for component in $(jq -r 'keys[] | select(. != ".")' .release-please-manifest.json); do
-        echo "DEBUG - component $component starting";
         component_name=$(echo "$component" | cut -d'/' -f2);
-        echo "DEBUG - component_name- $component_name";
         RELEASE_PLEASE_PR=$(gh pr list --author "${GITHUB_USER_NAME}" --head "release-please--branches--${CI_COMMIT_REF_NAME}" --search "${component_name}" --json number --jq '.[0].number');
         if [[ -z "$RELEASE_PLEASE_PR" ]]; then
-          echo "DEBUG - no PR found for $component_name";
+          echo "INFO - no PR found for $component_name";
           continue;
         fi
-        echo "DEBUG - component $component, component_name $component_name, RELEASE_PLEASE_PR $RELEASE_PLEASE_PR";
         gh pr checkout --force $RELEASE_PLEASE_PR;
         cd $component;
           wget --output-document cliff.toml https://raw.githubusercontent.com/mendersoftware/mendertesting/master/utils/cliff.toml.scoped;
           git cliff --unreleased --bump --output ./CHANGELOG.md --repository "../../" --include-path "${component}/**/*" --github-repo ${GITHUB_REPO_URL} --tag-pattern "${component_name}.*";
```

Comment:
wouldn't the `--unreleased` removal also be required here?

Reply:
yes! Good catch